test/authz-filler/config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "Filler": {
+    "Parallelism": 20,
+    "DBConnect": "mysql+tcp://sa@boulder-mysql:3306/boulder_sa_integration?readTimeout=14s&writeTimeout=14s&timeout=100ms"
```

Comment:
Can you use a `dbConnectFile` here instead and provide a path to a `DBConnect` value stored in `test/secrets/` similar to how the authz purger and other tools do it? That way there's only one place to update if we decide to change anything about the database authentication or connection parameters.